### PR TITLE
Added possible values for table frame attribute

### PR DESCRIPTION
**Pull request by EzioMercer**

### CODE REVIEW

**Comment by estelle** [comment]
Thank you for the contribution. The `frame` attribute of the `<table>` element is deprecated. The spec we follow on MDN is the WHATWG HTML living standard. https://html.spec.whatwg.org/multipage/tables.html#the-table-element. (Your link is HTML 4.01). Tables only have global attributes. styling should be done with CSS (separation of concerns), so all appearance related attributes are deprecated. 

Here is content that may be helpful for your students:

Styling tables:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables
https://web.dev/learn/html/tables/

Marking up tables:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#see_also
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Styling_tables

**Comment by EzioMercer** [comment]
I didn't understand your point. The `rules` attribute is also deprecated but you sepcify the possible values for this attribute. Why not do it for the `frame` attribute? What is difference between these two deprecated attributes?

**Comment by EzioMercer** [comment]
@estelle I didn't understand your point. The [rules](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#rules) attribute is also deprecated but you specify the possible values for this attribute. Why not do it for the [frame](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#frame) attribute? What is difference between these two deprecated attributes?

**Comment by estelle** [comment]
you have a very valid point. We should create an issue to determine whether we should be adding all the deprecated enumerated values that are missing (your PR), or removing the deprecated values that should not be used in favor of using CSS. The CSS comments are there, which is good, but unclear why we are including enumerated values for some deprecated attributes but not others.

**Comment by estelle** [comment]
Let's also change `rules` to read:

```md
This enumerated attribute defines where rules, i.e. lines, should appear in a table. Lines can be set to appear between `rows`, `cols` (for columns), or `groups` (between  `<thead>`, `<tbody>`, and `<tfoot>` elements, and between `<col>` and `<colgroup>` elements), or `all` or `none` of the above.
```

**Comment by EzioMercer** [comment]
@estelle Can you please help me? I can't understand how to update the PR preview page. I made changes which you asked but can't see the result

File: files/en-us/web/html/element/table/index.md [hunk 1]
```diff
@@ -119,19 +119,23 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
...
+
+    - `void`, which indicates that no sides will be displayed; it is the default value;
+    - `above`, which indicates that the top side only will be displayed;
+    - `below`, which indicates that the bottom side only will be displayed;
+    - `hsides`, which indicates that the top and bottom sides only will be displayed;
+    - `vsides`, which indicates that the right and left sides only will be displayed;
+    - `lhs`, which indicates that the left-hand side only will be displayed;
+    - `rhs`, which indicates that the right-hand side only will be displayed;
+    - `box`, which indicates that the all four sides will be displayed;
+    - `border`, which indicates that the all four sides will be displayed; it is same as `box`;
```
Comment (estelle) [inline files/en-us/web/html/element/table/index.md:132]:
```suggestion
  - : This enumerated attribute specifies which outer sides of the table should have borders drawn. The default `void` means no borders. The `above`, `below`, `lhs` (left-hand side), and `rhs` (right-hand side) lead to a border on a single side. Two sides will have borders with `hsides` (horizontal sides), `vsides` (vertical sides). All four sides will have a border if `box` or `border` is set. The CSS {{cssxref("border-style")}} property, set on the table, supersedes borders set by this HTML attribute.
```

File: files/en-us/web/html/element/table/index.md [hunk 2]
```diff
@@ -119,19 +119,23 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
...
 - `rules` {{Deprecated_inline}}
 
-  - : This enumerated attribute defines where rules, i.e. lines, should appear in a table. It can have the following values:
-
-    - `none`, which indicates that no rules will be displayed; it is the default value;
-    - `groups`, which will cause the rules to be displayed between row groups (defined by the {{HTMLElement("thead")}}, {{HTMLElement("tbody")}} and {{HTMLElement("tfoot")}} elements) and between column groups (defined by the {{HTMLElement("col")}} and {{HTMLElement("colgroup")}} elements) only;
-    - `rows`, which will cause the rules to be displayed between rows;
-    - `cols`, which will cause the rules to be displayed between columns;
-    - `all`, which will cause the rules to be displayed between rows and columns.
+  - : This enumerated attribute defines where rules, i.e. lines, should appear in a table. Lines can be set to appear between `rows`, `cols` (for columns), or `groups` (between `<thead>`, `<tbody>`, and `<tfoot>` elements, and between `<col>` and `<colgroup>` elements), or `all` or `none` of the above.
```
Comment (estelle) [inline files/en-us/web/html/element/table/index.md:138]:
```suggestion
  - : This enumerated attribute defines where rules, i.e. lines, should appear in a table. Lines can be set to appear between `rows`, `cols` (for columns), or `groups` (between `<thead>`, `<tbody>`, and `<tfoot>` elements, and between `<col>` and `<colgroup>` elements), or `all` or `none` of the above. CSS {{cssxref("border-style")}} supersedes borders set by this attribute.
```

**Review by estelle — COMMENTED** [review]
Suggested some edits

**Comment by estelle** [comment]
> @estelle Can you please help me? I can't understand how to update the PR preview page. I made changes which you asked but can't see the result

https://pr28399.content.dev.mdn.mozit.cloud/en-US/docs/Web/HTML/Element/table

the preview feature was down for a few days, so maybe it wasn't showing, but generally search in this page for "Preview URLs" and it should direct you to the github action creating a preview link

**Comment by EzioMercer** [comment]
@estelle Yes I know about that links but I can't understand how to update them and/or when do they updates automatically?

As you can see [here](https://github.com/mdn/content/pull/28399/commits/bd8f52126d1fe9bf7d1b6ab86808bb3f21363070) I already made changes for `rules` attribute but the preview link still shows old version of this page

image

So I'm a little confused. I thought that [this page](https://pr28399.content.dev.mdn.mozit.cloud/en-US/docs/Web/HTML/Element/table) will update after each commit

**Comment by bsmth** [comment]
Hi @EzioMercer - there are some review comments above waiting on you. Do you plan to come back to this one?

**Comment by EzioMercer** [comment]
Hi @bsmth! Of course I will, I'm waiting the response to my last question. I can't update the preview link to see how my last changes look like

**Comment by bsmth** [comment]
> Hi @bsmth! Of course I will, I'm waiting the response to my last question. I can't update the preview link to see how my last changes look like

I understand 👍🏻. You can check this sticky comment: https://github.com/mdn/content/pull/28399#issuecomment-1724694468 -  It should have the latest changes from this branch. I can see your proposed changes there as expected.

**Comment by bsmth** [comment]
It looks like this PR has stalled. What's next steps here @EzioMercer and @estelle?

**Comment by bsmth** [comment]
Hi both 👋🏻 I am going to close this one as it's stalled with conflicts & suggestions etc., additionally with the context of adding more info to deprecated features when there [are suggested alternatives](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#frame), I don't think we need to keep improving this section. Thank you! :)

**Pull request closed without merging**